### PR TITLE
Fix spawn messages linger when you leave the spawn queue and become a spectator.

### PR DESCRIPTION
**Pull request by DolceTriade**

### CODE REVIEW

**Review by necessarily-equal — APPROVED** [review]
LGTM, haven't tested

File: src/cgame/cg_rocket_draw.cpp [hunk 1]
```diff
@@ -2376,6 +2376,50 @@ class SpawnQueueElement : public TextHudElement
...
+			SetText( va( P_( "There is %d spawn remaining", "There are %d spawns remaining", this->spawns ), this->spawns ) );
+		}
+
+		if ( !IsVisible() )
+		{
+			SetProperty( Rml::PropertyId::Visibility, Rml::Property( Rml::Style::Visibility::Visible ) );
+		}
+	}
+
+	int spawns;
```
Comment (slipher) [inline src/cgame/cg_rocket_draw.cpp:2419]: We decided to go with trailing underscore for class members. https://wiki.unvanquished.net/wiki/Coding_convention#Naming_conventions

**Review by slipher — COMMENTED** [review]
LGTM other than style nit

File: src/cgame/cg_rocket_draw.cpp (hunk 1 above)
Comment (DolceTriade) [reply src/cgame/cg_rocket_draw.cpp:2419]: Ah I didn't realize that. I was sticking to the Unv convention of no identifier for no class members :D.

Will fix and merge later.

Comment (necessarily-equal) [reply src/cgame/cg_rocket_draw.cpp:2419]: Fort he record I don't think this is a problem because it is consistent with the rest of the file.

Comment (DolceTriade) [reply src/cgame/cg_rocket_draw.cpp]: Done.

**Comment by necessarily-equal** [comment]
👍

**Pull request merged**